RunTestsInBrowser.ps1

```diff
@@ -0,0 +1,10 @@
+# This script starts IIS Express and opens the Tests.html in a browser to host QUnit test runner
+# - Run this script: > powershell "& .\RunTestsInBrowser.ps1"
+# - Build the solution after changing TypeScript source files to regenerate JavaScript 
+# - Refresh Tests.html in the browser to run the JavaScript tests 
+
+$iisExpress = "C:\Program Files (x86)\IIS Express\iisexpress.exe"
```

Comment:
Good idea. Forced me to learn how to access environment variables in PowerShell :smile:

`${Env:ProgramFiles(x86)}`